Dockerfile

```diff
@@ -1,16 +1,22 @@
-FROM node:10
+FROM node:14 AS builder
 
 WORKDIR /usr/src/app
 
-RUN apt-get update && apt-get install -y vim secure-delete && apt-get clean
-
 COPY package*.json ./
 
 RUN npm install
 
-COPY . .
+FROM node:14-alpine
+
+WORKDIR /usr/src/app
+
+COPY config config
```

Comment:
I wonder if we need to copy this in as I would expect you'd want to mount the config in?

Reply:
Yeah, this seems odd. But that's what we do currently, so :shrug:

Reply:
I think it's only because of the tests - the default config is pretty much useless otherwise. But since the `make test` command requires the file to be there for now, I'd suggest we leave it and clean later.